seedorf/users/models.py

```diff
@@ -139,12 +141,20 @@ class MagicLoginLink(BasePropertiesModel):
         verbose_name=_("Magic login link"),
     )
 
-    token = models.CharField(
-        blank=False, null=False, max_length=32, verbose_name=_("Token"), unique=True, default=random_string
```

Comment:
@Tommos0 : we should remove the random_string function if its not being used anywhere else.

Reply:
Done, changed the old migration defaults to "dummy", that should be fine right?